nixos/modules/programs/command-not-found/rust/src/main.rs

```diff
@@ -0,0 +1,52 @@
+use rusqlite::{params, Connection, Result};
+use std::env;
+use std::process::exit;
+
+pub const NIX_SYSTEM: &str = env!("NIX_SYSTEM");
+pub const DB_PATH: &str = env!("DB_PATH");
```

Comment:
```suggestion
const NIX_SYSTEM: &str = env!("NIX_SYSTEM");
const DB_PATH: &str = env!("DB_PATH");
```
`pub` is not needed here.

(And I need to learn not forget to click the "add comment" button :D)